### PR TITLE
Create web views

### PR DESCRIPTION
**Pull request by ghost**
Created a basic web view listing all reports with the ability to filter by patient, health worker, and report status as well as order by any column.  Reports can also be exported to CSV.

### CODE REVIEW

File: nutrition/tests/views.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+from ..views import CSVNutritionReportList, NutritionReportList
+from ..models import Report
+from .base import NutritionTestBase
+
+
+__all__ = ['NutritionReportListViewTest', 'NutritionReportExportViewTest']
+
+
+class NutritionReportListViewTest(NutritionTestBase):
```
Comment (mlavin) [inline nutrition/tests/views.py]: Is this PR incomplete or are you not sure what to test here?

Comment (ghost) [inline nutrition/tests/views.py]: Incomplete.

File: nutrition/views.py [hunk 2]
```diff
@@ -1 +1,146 @@
...
+
+class NutritionReportMixin(object):
+    """Allow filtering by patient and healthworker."""
+    # Required permissions, in the format app_label.codename.
+    permissions = ['nutrition.view_report']
+
+    # Default order by which reports should be displayed.
+    ordering = ['-created']
+
+    def dispatch(self, request, *args, **kwargs):
```
Comment (mlavin) [inline nutrition/views.py:21]: Kind of a minor point but there is a built in decorator which already does this: https://docs.djangoproject.com/en/1.5/topics/auth/default/#the-permission-required-decorator

**Comment by mlavin** [comment]
My comment is nothing to hold back this change :sparkles:

**Pull request merged**